Classes/Command/InstallCommandController.php

```diff
@@ -326,14 +327,16 @@ private function ensureInstallationPossible($nonInteractive, $force)
                 $this->quit(2);
             }
         }
-        @unlink($localConfFile);
+        @unlink(is_link($localConfFile) ? readlink($localConfFile) : $localConfFile);
         @unlink($packageStatesFile);
         clearstatcache();
-        if (file_exists($localConfFile)) {
+        if (@is_file($localConfFile)) {
             $this->outputLine();
             $this->outputLine('<error>Unable to delete configuration file!</error>');
             $this->outputLine('<error>Installation aborted!</error>');
             $this->quit(3);
         }
+        //prewrite file to protect against the check in ConfigurationManager.php:273
+        file_put_contents($localConfFile, "<?php return [];");
```

Comment:
this breaks installation, as first install step will be skipped. what is this needed for anyway?